### ISSUE TITLE
Reshape provinces into something more organic

### ISSUE THREAD

**Issue by Araxiel** [Araxiel/Dead_Money-A_Vicky3_Fallout_Mod#112]
The province generation used fairly basic voronoi with straight lines. While usable, it looks far from great.
Vanilla provinces are much, much more organic looking.
![image](https://github.com/Araxiel/Dead_Money-Vicky3_Fallout/assets/11295969/eb86793b-875d-4148-a48e-c6fe7ff7fddf)
![image](https://github.com/Araxiel/Dead_Money-Vicky3_Fallout/assets/11295969/83cf0cde-a3ae-460a-97be-7a5aa04f9cc0)

TLDR, the provinces need to _squiqqlyfied_
